### PR TITLE
[Blog][Ready to review] Meshery Playground: Features you don't want to miss

### PR DESCRIPTION
**Pull request by saurabh100ni**
**Description**

I wrote blog on Meshery Playground which is ready for reviews.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f2e0633096b976efd1800d2f73a5111ed5a164e9 at: https://655cc4fe18276d035af13ffa--layer5.netlify.app

File: src/collections/blog/2023/11-15-Features-of-meshery/index.mdx [hunk 1]
```diff
@@ -0,0 +1,185 @@
+---
+title: "Meshery Playground"
+subtitle: "Meshery Playground: Features you don't want to miss"
+date: 2023-11-15 10:30:05 -0530
+author: Saurabh Soni
+thumbnail: "./MesheryLogo.webp"
+darkthumbnail: "./MesheryLogo.webp"
+
+category: "Meshery"
+description: "Features of Meshery Playground which takes your user experience to the next level."
```
Comment (leecalcote) [inline src/collections/blog/2023/11-15-Features-of-meshery/index.mdx:10]:
```suggestion
description: "Key features of Meshery Playground for Kubernetes, Prometheus, Argo and other CNCF projects. Use Meshery Playground to explore a new way of DevOps - visual and collaborative configuration management for your infrastructure. "  
```

File: src/collections/blog/2023/11-15-Features-of-meshery/index.mdx [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+resource: false
+published: true
+---
+
+import { BlogWrapper } from "../../Blog.style.js";
+import mesheryLogo from "../../../../assets/images/meshery/icon-only/meshery-logo-light.svg";
+
+<BlogWrapper>
+
+Before we dive into the features, Let’s take a recap, What is [Meshery Playground](https://playground.meshery.io/)?
```
Comment (leecalcote) [inline src/collections/blog/2023/11-15-Features-of-meshery/index.mdx:23]:
```suggestion
Before we dive into features, let’s understand what the Meshery Playground is.
```

**Comment by leecalcote** [comment]
There are a number of inconsistencies to cleanup. Please do so.

**Comment by l5io** [comment]
🚀 Preview for commit b9ddb949ff00805cecdd1340491061d3324272f4 at: https://655dac4601df1a624d2f24c7--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e8f49355b3fc8e6c30d9001f2055b151da8071c5 at: https://655db36f9cde316d62a65bef--layer5.netlify.app

**Comment by saurabh100ni** [comment]
@leecalcote Done fixing the inconsistency with the formatting and links and also added your suggestions.

File: src/collections/blog/2023/11-15-Features-of-meshery/index.mdx [hunk 3]
```diff
@@ -143,7 +143,7 @@ Mesheryctl supports multiple profiles, allowing users to switch between differen
 Mesheryctl supports subcommands for various commands, providing a convenient way to interact with Meshery Server.
 
 # [Meshery UI](https://meshery.io/):
-Meshery UI is a web-based user interface for Meshery, offering a convenient way to interact with Meshery Server. Meshery UI is a valuable tool for DevOps engineers, enhancing the efficiency of their cloud-native development and operations workflows. Key features include:
+[Meshery UI](https://meshery.io/) is a web-based user interface for Meshery, offering a convenient way to interact with Meshery Server. Meshery UI is a valuable tool for DevOps engineers, enhancing the efficiency of their cloud-native development and operations workflows. Key features include:
```
Comment (leecalcote) [inline src/collections/blog/2023/11-15-Features-of-meshery/index.mdx:146]: Too many links.

File: src/collections/blog/2023/11-15-Features-of-meshery/index.mdx [hunk 4]
```diff
@@ -152,9 +152,9 @@ Meshery UI allows users to configure their Meshery Server, providing a convenien
 Meshery UI supports multiple contexts, allowing users to switch between different Meshery Server instances.
 
 
-# [Meshery catalog](https://meshery.io/catalog):
+# [Meshery catalog](https://docs.meshery.io/concepts/catalog):
```
Comment (leecalcote) [inline src/collections/blog/2023/11-15-Features-of-meshery/index.mdx:155]: Why is catalog not capitalized?

File: src/collections/blog/2023/11-15-Features-of-meshery/index.mdx [hunk 5]
```diff
@@ -176,6 +176,7 @@ Open a terminal or command prompt and use Meshery CLI commands to interact with
 Commands include onboarding an application, applying a WASM filter, viewing, deleting, and listing patterns.	
 Meshery Catalog, with its user-friendly approach and collaborative features, stands as a valuable resource for cloud-native practitioners, facilitating seamless deployment, knowledge-sharing, and the creation of enhanced design patterns within the Meshery ecosystem.
```
Comment (leecalcote) [inline src/collections/blog/2023/11-15-Features-of-meshery/index.mdx:177]: This is a poor description of the Catalog. This is a description of Meshery.

File: src/collections/blog/2023/11-15-Features-of-meshery/index.mdx [hunk 6]
```diff
@@ -176,6 +176,7 @@ Open a terminal or command prompt and use Meshery CLI commands to interact with
...
 
+# Conclusion:
 In summary, Meshery empowers teams through its Workspaces feature, providing a virtual space for seamless collaboration, resource sharing, and efficient management of diverse environments. With logical grouping, flexible support for various resources, and robust access control, Workspaces serve as a centralized hub for team-based work. By adhering to best practices, such as clearly defined permissions and leveraging Infrastructure Designs, users can fully harness the collaborative potential of Meshery Workspaces. Maximize your team's efficiency and streamline your cloud-native development processes with Meshery Workspaces.
```
Comment (leecalcote) [inline src/collections/blog/2023/11-15-Features-of-meshery/index.mdx:180]: In summary, Meshery boils down to its Workspace feature? Meshery doesn't even have this feature, yet. Cloud does.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit b96b7023ac2aea8915312cf5162bae5cd956d599 at: https://655e0bb9e48e773042716763--layer5.netlify.app

**Pull request closed without merging**